Cargo.lock

```diff
@@ -2457,12 +2489,30 @@ dependencies = [
  "libc",
 ]
 
+[[package]]
+name = "semver"
```

Comment:
Did you run `cargo update`?

Reply:
no, the Cargo.lock changes should just be due to the new dependency added.